src/components/AttachmentModal.js

```diff
@@ -306,7 +327,12 @@ class AttachmentModal extends PureComponent {
                 {this.props.children({
                     displayFileInModal: this.validateAndDisplayFileToUpload,
                     show: () => {
-                        this.setState({isModalOpen: true});
+                        const route = Navigation.getActiveRoute();
+                        let reportId = null;
```

Comment:
rename to `reportID` as well. Similar comment with other places you use `reportId` instead of `reportID`